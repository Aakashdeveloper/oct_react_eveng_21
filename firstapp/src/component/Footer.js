import React from 'react';

const Footer = (props) => {
    // console.log(props)
    return(
        <>
            <hr/>
            <center>
                <h3>&copy; Developers Funnel {props.month} {props.year}</h3>
            </center>
        </>
    )
}

export default Footer;