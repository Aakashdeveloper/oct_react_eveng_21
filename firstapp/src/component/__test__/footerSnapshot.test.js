import React from 'react';
import Footer from '../Footer';
import {create} from 'react-test-renderer';

describe('Snapshot test',() => {
    test('testing footer',() => {
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})