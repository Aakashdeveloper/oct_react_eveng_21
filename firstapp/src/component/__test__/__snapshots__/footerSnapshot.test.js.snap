// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test testing footer 1`] = `
Array [
  <hr />,
  <center>
    <h3>
      Â© Developers Funnel 
       
    </h3>
  </center>,
]
`;
